README.md

```diff
@@ -23,20 +23,8 @@ Each top level folder in `src/js` has its own README with more specific informat
 ## How to use
 Usage instructions can be found [here](./USAGE.md)
 
-## Demo
-### Steps for testing:
+## How to test
```

Comment:
Under How to test section, please add "Instructions for testing the changes can be found [here](./Demo.md)"